pkg/apis/submariner.io/v1/types.go

```diff
@@ -19,6 +19,7 @@ type ClusterSpec struct {
 	ColorCodes  []string `json:"color_codes"`
 	ServiceCIDR []string `json:"service_cidr"`
 	ClusterCIDR []string `json:"cluster_cidr"`
+	GlobalCIDR  []string `json:"global_cidr"`
```

Comment:
At some point we will need to switch globalCidr may be, since the "_" format is not OpenAPI compliant, not sure if globalCIDR... we may need to check. 

But for now it's better to keep coherence as you did.